tests/bazel_run_tests.sh

```diff
@@ -16,61 +16,56 @@ function test_duplicate_version_warning() {
 }
 
 function test_m2local_testing_found_local_artifact_through_pin() {
-  m2local_dir="~/.m2/repository"
+  m2local_dir="${HOME}/.m2/repository"
```

Comment:
Perhaps this should be the result of `mktemp` instead of a user's home directory. I can see this having hilarious unforeseen consequences.

Reply:
Ok, it was working locally because I still had the kt example lib in my `~/.m2/repository` directory.  I changed it so it still goes to local but it only removes the directory for the kt example jar instead.